recipes/htsinfer/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "HTSinfer" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/zavolanlab/htsinfer/archive/refs/tags/v0.9.0.tar.gz
+  sha256: f4708d43e56389240752e45ac3f4cdcb620ab3783b57ff068599eec0adbd91f3
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - htsinfer = htsinfer.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv"
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin="x.x") }}
+
+requirements:  
+  host:
+    - python >=3.8, <=3.10
+    - pip >=20.2.3
+    - setuptools-git >=1.2
+    - numpy >=1.22, <1.25
+    - pandas >=1.3.5, <1.4.0
+    - biopython >=1.78
+    - pydantic >=1.8.1, <2
+    - pysam >=0.16.0
+    - pyahocorasick >=1.4.0
+  run:
+    - biopython >=1.78
+    - cutadapt >=3.5, <=4.2
+    - kallisto >=0.46.1, <= 0.48.0
+    - numpy >=1.22, <1.25
+    - pandas >=1.3.5, <1.4.0
+    - pip >=20.2.3
+    - pyahocorasick >=1.4.0
+    - pydantic >=1.8.1, <2
+    - pysam >=0.16.0
+    - python >=3.8, <=3.10
+    - star >=2.7.6
+
+test:
+  commands:
+    - htsinfer --version
+
+about:
+  home: https://github.com/zavolanlab/htsinfer
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: HTSinfer infers metadata from Illumina high throughput sequencing (HTS) data
```

```suggestion
    - pyahocorasick >=1.4.0
    - pydantic >=1.8.1, <2
    - pysam >=0.16.0
    - python >=3.8, <=3.10
    - star >=2.7.6

test:
  commands:
    - htsinfer --version

about:
  home: https://github.com/zavolanlab/htsinfer
  license: Apache-2.0
  license_family: APACHE
  license_file: LICENSE
  summary: HTSinfer infers metadata from Illumina high throughput sequencing (HTS) data

extra:
  recipe-maintainers:
    - balajtimate
```
Minor suggestion to remove `pip` from the list of run dependencies (it is only needed during build time) and add yourself as the `recipe-maintainer`, otherwise LGTM!